harshita/level1/examallowedornot.cpp

```diff
@@ -0,0 +1,21 @@
+#include<iostream>
+using namespace std;
+
+
+int main()
+{
+	int numclass, attend, total;
+	cout << "Number of Classes: ";
+	cin >> numclass;
+	cout << "Number of Classes attended: ";	
+	cin >> attend;
+	total= (attend / numclass)* 100;
```

Comment:
did you compile and check the result ?? See lets say we have numclass = 10 & attend = 7. then  attend/numclass = 7/10 = 0 as it will give int as ans. So we need to do typecasting. `((float)attend / numclass)* 100` or you can do something like this `attend*100.0/numclass`;

Reply:
yes checked it